pom.xml

```diff
@@ -0,0 +1,117 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.tenx.ms</groupId>
+        <artifactId>microservice-parent</artifactId>
+        <version>LATEST</version>
+        <relativePath/>
+    </parent>
+
+    <groupId>com.tenx.ms</groupId>
+    <artifactId>homes-ms-retail</artifactId>
+    <packaging>jar</packaging>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>Retail Microservice</name>
+    <description>Retail MicroService</description>
+
+    <properties>
+        <start-class>com.tenx.ms.retail.RetailServiceApp</start-class>
+        <api-doc.basepath>/retail</api-doc.basepath>
+        <api-doc.version>v1</api-doc.version>
+        <api-doc.title>Retail Service</api-doc.title>
+        <api-doc.description>Provides Retail services</api-doc.description>
+        <api-doc.port>9990</api-doc.port>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.flywaydb</groupId>
+            <artifactId>flyway-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.data</groupId>
+            <artifactId>spring-data-jpa</artifactId>
+            <version>1.9.4.RELEASE</version>
```

Comment:
Spring Boot specifies a version for this artifact, so we would typically omit the `<version>` element and let Maven get that version. By the way, because `spring-boot-starter-data-jpa` already pulls in this artifact.